libraries/azure-app-configuration-importer/src/internal/parsers/defaultConfigurationSettingsConverter.ts

```diff
@@ -272,4 +375,45 @@ class FeatureFlagConfigurationSettingsConverter implements ConfigurationSettings
 
     return lowerCaseFilters;
   }
+
+  private validateRequirementType(requirement_type: any, featureFlagName: string) {
+    if(!Object.values(RequirementType).includes(requirement_type)){
+      throw new ArgumentError(
+        `This feature flag '${featureFlagName}' must have any/all requirement type.`
+      );
+    }
+
+    return true;
+  }
+
+  private getMsFmSchemaSection(config: object): any {
+    const featureManagementKey = this.msFmFeatureManagementKeyWord as keyof object;
+    const featureManagementSection = config[featureManagementKey];
+    if (featureManagementSection && typeof featureManagementSection === "object") {
+      if (!Array.isArray(featureManagementSection[Constants.FeatureFlagsKeyWord])) {
+        throw new ArgumentError(`The ${Constants.FeatureFlagsKeyWord} key within ${this.msFmFeatureManagementKeyWord} must be an array.`);
+      }
+      return featureManagementSection[Constants.FeatureFlagsKeyWord];
+    }
+  }
+
+  private getLegacySchema(config: object): any {
+    if (this.legacySchemaFeatureManagementKeyWord) {
```

Comment:
Don't check in this method, we check outside according to "foundLegacySchema"

Reply:
Checked outside